File: 231-power-of-two/solution.py
# Runtime: 35ms, Beats 69.97% of users with Python3
# Memory: 16.47MB, Beats 96.02% of users with Python3

class Solution:
    def isPowerOfTwo(self, n: int) -> bool:
        return n>0 and n & (n-1) == 0

File: 231-power-of-two/solution.cpp
// Runtime: 35ms, Beats 100.00% of users with C++
// Memory: 16.47MB, Beats 7.34% of users with C++

class Solution {
public:
    bool isPowerOfTwo(int n) {
        return n>0 && (n & (n-1)) == 0;
    }
};